NineChronicles.Headless.AccessControlCenter/Controllers/AccessControlServiceController.cs

```diff
@@ -36,6 +37,21 @@ public ActionResult AllowAccess(string address)
             return Ok();
         }
 
+        [HttpPost("entries/add-tx-quota/{address}/{quota}")]
```

Comment:
```suggestion
        [HttpPost("entries/add-tx-quota/{address}/{quota:int}")]
```

Reference: https://learn.microsoft.com/en-us/aspnet/core/mvc/controllers/routing?view=aspnetcore-7.0#attribute-routing-with-http-verb-attributes